opencog/spacetime/TimeSpaceAtomMapsAPI/TestTimeSpaceAtom.cpp

```diff
@@ -0,0 +1,167 @@
+/*
+ *
+ * Copyright (c) 2016, Mandeep Singh Bhatia, OpenCog Foundation
+ * All rights reserved.
+ * License: New BSD
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ *     * Redistributions of source code must retain the above copyright
+ *       notice, this list of conditions and the following disclaimer.
+ *     * Redistributions in binary form must reproduce the above copyright
+ *       notice, this list of conditions and the following disclaimer in the
+ *       documentation and/or other materials provided with the distribution.
+ *     * Neither the name of the University of Freiburg nor the names of its
+ *       contributors may be used to endorse or promote products derived from
+ *       this software without specific prior written permission.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+ * ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE
+ * LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+ * CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ * SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ * INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ * CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ * ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ * POSSIBILITY OF SUCH DAMAGE.
+ */
+ 
+//TestAtomOcTree.cpp
+#include "assert.h"
+#include "TimeSpaceAtom.h"
+
+using namespace std;
+using namespace octomap;
+
+
+void print_query_info(point3d query, aHandle ato) {
+  if (ato != UndefinedHandle) {
+    cout << "occupancy probability at " << query << ":\t " << ato<< endl;
+  }
+  else 
+    cout << "occupancy probability at " << query << ":\t is unknown" << endl;    
+}
+
+int main(int argc, char** argv) {
+
+  cout << endl;
```

Comment:
Hi Nil,
 Thanks for the review. I will make those changes. I just copy pasted the
license so I made a mistake, I will change the license to gpl.
Thanks,
Mandeep

On Tue, Mar 8, 2016 at 9:02 AM, ngeiswei <notifications@github.com> wrote:

> In opencog/spacetime/TimeSpaceAtomMapsAPI/TestTimeSpaceAtom.cpp
> <https://github.com/opencog/opencog/pull/2017#discussion_r55301380>:
>
> > +
> > +using namespace std;
> > +using namespace octomap;
> > +
> > +
> > +void print_query_info(point3d query, aHandle ato) {
> > +  if (ato != UndefinedHandle) {
> > +    cout << "occupancy probability at " << query << ":\t " << ato<< endl;
> > +  }
> > +  else
> > +    cout << "occupancy probability at " << query << ":\t is unknown" << endl;
> > +}
> > +
> > +int main(int argc, char** argv) {
> > +
> > +  cout << endl;
>
> Your indentation scheme is a bit inconsistent. Please try to follow
>
> http://wiki.opencog.org/w/Development_standards
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/opencog/opencog/pull/2017/files#r55301380>.
>